src/test/user.spec.js

```diff
@@ -193,3 +200,226 @@ describe('Test user login, signup and account verification', () => {
     });
   });
 });
+
+describe('USER CAN FOLLOW OTHER USERS', () => {
+  beforeEach(async () => {
+    await db.User.destroy({ cascade: true, truncate: true });
+  });
+
+  it('should be able to follow other users', async () => {
```

Comment:
test case description should read as continuation of test block description

Reply:
changes effected, thank you